### PR TITLE
Create UI List Movie

### PR DESCRIPTION
**Pull request by VanHieu7119**
Create UI Screen List Movie
Link Redmine: https://edu-redmine.framgia.vn/issues/3715
image

### CODE REVIEW

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
-        <activity android:name=".MainActivity">
+        <activity android:name=".screen.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
 
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity android:name=".screen.listMovie.ListMovieActivity">
+        </activity>
```
Comment (tuanlvt-1117) [inline app/src/main/AndroidManifest.xml]: dùng thẻ tự đóng

File: app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieAdapter.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        public ObservableField mName = new ObservableField();
+        public ObservableField mRating = new ObservableField();
+        public ObservableField mDate = new ObservableField();
+        public ObservableField mPoster = new ObservableField();
+        private ItemListMovieBinding mItemListMovieBinding;
+
+        ViewHolder(@NonNull ItemListMovieBinding itemListMovieBinding) {
+            super(itemListMovieBinding.getRoot());
+            this.mItemListMovieBinding = itemListMovieBinding;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieAdapter.java]: remove this

File: app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieViewModel.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    private void setData() {
+        List<Movie> movies = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            Movie movie = new Movie();
+            movie.setNameMovie("DEAD POOL");
+            movie.setDate("30/10/2018");
+            movie.setRating("8.2");
+            movie.setPoster(String.valueOf(R.drawable.poster2));
+            movie.setContent("Introduce Content Movie");
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/moviedb_31/screen/listMovie/ListMovieViewModel.java]: hard code

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**